apps/ledger-live-desktop/src/renderer/components/OnboardingAppInstall/CancelModal.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Button, Flex, Icons, Popin, Text } from "@ledgerhq/react-ui";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+
+const AppInstallPopin = styled(Popin)`
+  max-height: unset !important;
+  height: unset !important;
+`;
+
+type Props = {
+  isOpen: boolean;
+  productName: string;
+  onRetry?: () => void;
+  onSkip?: () => void;
```

Comment:
`onRetry` and `onSkip` don't have to be optional

Reply:
Done